gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -196,13 +197,13 @@ public synchronized void createEmptyObjects(
   }
 
   @Override
-  public SeekableByteChannel open(
+  public GoogleCloudStorageReadableByteChannel open(
```

Comment:
Since we are changing signatures of public interface, how are we planning to release it?

Reply:
`GoogleCloudStorageReadableByteChannel` still extends `SeekableByteChannel` and it will not be a breaking change for existing users. They although will not be able to utilize the new APIs unless they start consuming `GoogleCloudStorageReadableByteChannel`.

Reply:
only other alternate is to add new API something like `openWithVectored`